Cargo.toml

```diff
@@ -7,14 +7,15 @@ license = "Apache-2.0/MIT"
 name = "addr2line"
 readme = "./README.md"
 repository = "https://github.com/gimli-rs/addr2line"
-version = "0.1.0"
+version = "0.2.0"
 
 [dependencies]
 fallible-iterator = "0.1.3"
-getopts = "0.2.14"
+clap = "2.*"
 gimli = "0.9.0"
 memmap = "0.5.0"
 object = "0.1.0"
+owning_ref = { git = "https://github.com/Kimundi/owning-ref-rs" } # need OwningHandle
```

Comment:
This is just a placeholder until the next release is cut? We won't be able to publish on crates.io with a git dependency :-/